hybrid/HybridSampleApps/AccountEditor/AndroidManifest.xml

```diff
@@ -35,7 +35,7 @@
             'scheme', 'host' and 'path' with their corresponding values from your connected app.
 
             For example, if the callback URL of your connected app is
-            "testsfdc:///mobilesdk/detect/oauth/done",
+            "https://login.salesforce.com/services/oauth2/success",
```

Comment:
Just for reference I would not advise using a URL (especially this URL) as a the callback URI.  If you need to use Advanced (browser) Auth the callback is used to deep link back into your app and Google requires proof of ownership for URLs.  There is no way to prove you own `https://login.salesforce.com`.